core/js/pokemon.maps.js

```diff
@@ -420,3 +442,29 @@ function isMobileDevice() {
     //  Basic mobile OS (not browser) detection
     return (/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent))
 }
+
+function coordinate_area(){
+	$.ajax({
+		'async': false,
```

Comment:
I can try it. I needed synch load because we need the coordinates before the map is loading.